native/common/jp_method.cpp

```diff
@@ -123,7 +123,14 @@ JPMethodOverload* JPMethod::findOverload(vector<HostRef*>& arg, bool needStatic)
 	if (!maximallySpecificOverload)
 	{
 		std::stringstream ss;
-		ss << "No matching overloads found for " << getName() << " in find.";
+		ss << "No matching overloads found for " << getName() << ", options are:" << std::endl;
+		for (std::vector<OverloadData>::iterator it = m_OverloadOrderCache.begin();
+				it != m_OverloadOrderCache.end();
+				++it)
+		{
+				ss << it->m_Overload->getSignature();
+				ss << std::endl;
+		}

```

Comment:
Similar to CHANGELOG.rst, this is the only real change in jp_method.cpp.  The other changed lines in this file are from the removal of trailing whitespaces.